rewrite-maven/build.gradle.kts

```diff
@@ -67,6 +67,8 @@ dependencies {
     testRuntimeOnly("org.mapdb:mapdb:latest.release")
     testRuntimeOnly(project(":rewrite-java-11"))
     testRuntimeOnly("org.rocksdb:rocksdbjni:latest.release")
+    testRuntimeOnly("org.apache.logging.log4j:log4j-api:latest.release")

```

Comment:
added them for the unit tests for a more realistic test of adding a bom.
they could be removed I would just have to rewrite the tests.. 🤔 perhaps a depends on since the tests only have a single use of the log4j

Reply:
that worked replace the dependencies with stubs for the test parser